package.json

```diff
@@ -31,9 +31,11 @@
   "dependencies": {
     "@adobe/create-helix-library": "1.4.6",
     "@adobe/helix-epsagon": "1.6.0",
+    "@adobe/helix-fetch": "^2.1.7",
```

Comment:
pin

hint: when you use `npm add -E @adobe/helix-fetch`, it uses the exact version....

Reply:
```suggestion
    "@adobe/helix-fetch": "2.1.7",
```